community/lld/build

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+cd lld
```

Comment:
https://k1sslinux.org/wiki/kiss/style-guide#0800

This would end up looking something like
```
cmake -B build \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DCMAKE_BUILD_TYPE=Release \
    -Wno-dev

cmake --build   build
cmake --install build
```

If we have to go this route (see my other comment), the last line would read
`-Wno-dev lld`

Reply:
Done that, will force push with other changes in a sec!